applications/vanilla/controllers/api/CategoriesApiController.php

```diff
@@ -219,7 +225,8 @@ public function get_search(array $query) {
             $query['query'],
             $query['expand'],
             $limit,
-            $offset
+            $offset,
+            ($query['expand'] ?? true) ? ['breadcrumbs'] : []
```

Comment:
The description on the expand parameter say it's to "expand with the parent record", but that isn't accurate any more. I recommend altering the expand parameter definition to be built with the `\Vanilla\ApiUtils::getExpandDefinition` method. This will allow you to use specific fields (e.g. "parent" and "breadcrumbs") and should be backwards-compatible with the boolean expand this endpoint currently has. From there, you can use the controller's `isExpandField` method for explicitly expanding certain items.

Reply:
Transformation is done. Please check updated.